java-library/src/main/java/com/microsoft/azure/functions/openai/annotation/assistant/AssistantCreateRequest.java

```diff
@@ -62,4 +64,46 @@ public String getInstructions() {
     public void setInstructions(String instructions) {
         this.instructions = instructions;
     }
+
+    /**
+     * Gets the configuration section name for the table settings for assistant chat
+     * storage.
+     * 
+     * @return The configuration section name for the table settings for assistant
+     *         chat storage.
+     */
+    public String getChatStorageConnectionSetting() {
+        return chatStorageConnectionSetting;
+    }
+
+    /**
+     * Sets the configuration section name for the table settings for assistant chat
+     * storage.
+     * 
+     * @param chatStorageConnectionSetting The configuration section name for the
+     *                                     table settings for assistant chat
+     *                                     storage.
+     */
+    public void setChatStorageConnectionSetting(String chatStorageConnectionSetting) {
+        this.chatStorageConnectionSetting = chatStorageConnectionSetting;
+    }
+
+    /**
+     * Gets the table collection name for assistant chat storage.
+     * 
+     * @return The table collection name for assistant chat storage.
+     */
+    public String getCollectionName() {
```

Comment:
same for collectionName. Could this be a get/set property as well?

Reply:
yes, it is a get set property, both getter and setter are defined for it.
Looks like you are confusing it for C#, this is java library code.